include/drivers/st25r3911b_nfc.rst

```diff
@@ -0,0 +1,84 @@
+.. _st25r3911b_nfc_readme:
+
+NFC Reader ST25R3911B
+#####################
+
+NFC Reader ST25R3911B can be used to read/write in ISO/IEC 14443 A compatible NFC tags,
```

Comment:
Maybe better: "The NFC Device ST25R3911B can be used to read/write NFC-A compatible tags (tags which support ISO/IEC 14443 standard)".

Reply:
changed